src/lib/tvgPictureImpl.h

```diff
@@ -205,6 +219,13 @@ struct Picture::Impl
         if (!loader->read()) return Result::Unknown;
         w = loader->w;
         h = loader->h;
+        if (renderer) dispose(*renderer);
```

Comment:
Dismiss the renderer... a bad case in the point of performance wise.
That would be better if we can prevent this scenario ealier from the app.

Reply:
Ok. Removed disposing to just checking image is already loaded. If so, return InsufficientCondition.